src/main/java/com/javadiscord/javabot/commands/user_commands/Regex.java

```diff
@@ -0,0 +1,38 @@
+package com.javadiscord.javabot.commands.user_commands;
+
+import com.javadiscord.javabot.commands.ResponseException;
+import com.javadiscord.javabot.commands.SlashCommandHandler;
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.events.interaction.SlashCommandEvent;
+import net.dv8tion.jda.api.requests.restaction.interactions.ReplyAction;
+
+import java.util.regex.Pattern;
+
+public class Regex implements SlashCommandHandler {
+
+    @Override
+    public ReplyAction handle(SlashCommandEvent event) throws ResponseException {
+
```

Comment:
I know that those slash command options are set as 'required', but maybe add tiny checks for those options to make sure they're not null. 

https://github.com/Java-Discord/JavaBot/blob/main/src/main/java/com/javadiscord/javabot/commands/staff_commands/Warn.java#L71-L79